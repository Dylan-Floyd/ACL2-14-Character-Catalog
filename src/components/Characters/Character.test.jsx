import { render, screen } from '@testing-library/react'
import Character from './Character.jsx'

it('should match the snapshot', () => {
  const charData = {
    image: 'asdf.png',
    name: 'bob',
    species: 'man',
    status: 'alive'
  }
  const { container } = render(<Character character={charData} />)
  expect(container).toMatchSnapshot()
})