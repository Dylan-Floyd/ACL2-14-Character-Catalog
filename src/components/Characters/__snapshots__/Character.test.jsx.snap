// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div>
  <figure>
    <img
      alt="bob"
      src="asdf.png"
    />
    <figcaption>
      <h3>
        bob
      </h3>
      <p>
        man
      </p>
      <p>
        Status: 
        alive
      </p>
    </figcaption>
  </figure>
</div>
`;
